src/data/faq.json

```diff
@@ -51,10 +51,13 @@
                         "anchor": "error_log",
                         "active": true,
                         "textblock": [
-                            "With version 2.2, the Corona-Warn-App (CWA) offers a new function for detailed technical problem analysis. The error report is intended to help the developers to faster perform a root cause analysis and to eliminate errors more efficiently.<br><b> Important:</b> Please report an issue <b>first</b> to the technical hotline (national: 0800 7540001 or international: +49 30 498 75401) to verify if there is already a known solution. An error report is only required if the CWA team asks for this report.",
-                            "<ol><li>If so, please open a issue on GitHub and describe the problematic behavior as detailed as possible.</li><li>Start logging in the CWA (<a href='../blog/2021-05-12-corona-warn-app-version-2-2/' target='_blank' rel='noopener noreferrer'>see blog article</a>).<br>-Try to reproduce the faulty behavior or try to generate the error message.<br>- If this attempt was successful, send the error report to the CWA server. The data will be anonymized and encrypted. (If you are interested, you may also save the report locally on your Smartphone.)<br>- In return you will receive a unique ID, which you can also find in the list of all IDs.<br>- Stop the error report.</li><li>Update the GitHub issue with your unique identifier from the server so that the report can be linked to your issue.</li></ol>",
-                            "Remarks:<ul><li>Error reports are automatically deleted after 14 days.</li><li>Error reports that are transmitted to the server without request by the CWA team cannot be analyzed and processed.</li><li>There is no back channel into the Corona-Warn-App for the error reports. The CWA technical team can only communicate via GitHub to an issue.</li></ul>",
-                            "See also the blog article <a href='../blog/2021-05-12-corona-warn-app-version-2-2/' target='_blank' rel='noopener noreferrer'>Corona-Warn-App version 2.2 enables creation of a rapid test profile and error reports</a>."
+                            "With version 2.2, the Corona-Warn-App (CWA) offers a new function for detailed technical problem analysis. The error report is intended to help the developers to perform a root cause analysis faster and to eliminate errors more efficiently. <br><b>Important:</b> Please report an issue <b>first</b> to the technical hotline, either by phone (<a href='#international_phone_numbers' target='_blank' rel='noopener noreferrer'>click here to go see the phone numbers</a>) or by e-mail (<a href='mailto:corona-warn-app.opensource@sap.com'>corona-warn-app.opensource@sap.com</a>), to verify if there is already a known solution. An error report is only required if the technical support asks for this report.",

```

Comment:
technical hotline -> technical support

Reply:
Done in https://github.com/corona-warn-app/cwa-website/pull/1267/commits/880a279dd6a7d5db22940a8e9fb58b8e745b9174